### PR TITLE
Update to Agent 1.17.3

### PR DESCRIPTION
**Pull request by jhaynes**
### Summary
Cache Agent 1.17.3

### Licensing

This contribution is under the terms of the Apache 2.0 License:  yes

### CODE REVIEW

File: packaging/amazon-linux-ami/ecs-init.spec [hunk 1]
```diff
@@ -152,6 +152,8 @@ if [ -e %{running_semaphore} ]; then
 fi
 
 %changelog
+* Fri Mar 30 2018 Justin Haynes <jushay@amazon.com> - 1.17.3-1
+- Cache Agent vesion 1.17.3
```
Comment (aaithal) [inline packaging/amazon-linux-ami/ecs-init.spec]: I think https://github.com/aws/amazon-ecs-init/pull/160 warrants a changelog as well (as an `Enhancement`).

Comment (jahkeup) [reply packaging/amazon-linux-ami/ecs-init.spec]: Yep! I think we may want to adopt the `CHANGELOG.md` procedure that the agent follows and port that to package specific changelogs at release time.

Comment (jhaynes) [reply packaging/amazon-linux-ami/ecs-init.spec]: Agreed. For now, I'll add `- Use s3client instead of httpclient when downloading` to the spec file.

Comment (jahkeup) [reply packaging/amazon-linux-ami/ecs-init.spec]: Added an issue to track this thought: https://github.com/aws/amazon-ecs-init/issues/162

**Pull request merged**